travis/setup_dependencies_common.sh

```diff
@@ -336,6 +332,8 @@ if [[ $SETUP_CMD == *build_sphinx* ]] || [[ $SETUP_CMD == *build_docs* ]]; then
     # We don't want to install everything listed in the PIN_FILE in this section
 
     for package in matplotlib sphinx; do
+        # ensure pin file exists before moving it
+        touch $PIN_FILE
```

Comment:
It looks like this change means that the pin file isn't always present. Here I'm forcing it to be present. I didn't find anything in the manpage of `mv` to handle this more gracefully.

Reply:
I think it would make sense to have this at the top in L85 after the definition of ``$PIN_FILE``